lib/ex_unit/lib/ex_unit/server.ex

```diff
@@ -83,40 +81,22 @@ defmodule ExUnit.Server do
       %{config | captured_devices: Set.delete(config.captured_devices, device)}}
   end
 
-  def handle_call({:log_capture_on, pid, args}, _from, config) do
-    ref = Process.monitor(pid)
-    try do
-      GenEvent.add_mon_handler(Logger, {Console, ref}, args)
-    catch
-      :exit, :noproc ->
-        Process.demonitor(ref, [:flush])
-        {:reply, {:error, :no_logger}, config}
+  def handle_call({:log_capture_on, pid}, _from, config) do
+    ref  = Process.monitor(pid)
+    refs = HashSet.put(config.log_captures, ref)
+
+    if HashSet.size(refs) == 1 do
+      status = Logger.remove_backend(:console)
```

Comment:
We won't reach this if the Logger app is not running because add_capture would fail. :)